vnext/src/Libraries/Components/TextInput/TextInput.windows.js

```diff
@@ -852,6 +852,8 @@ function InternalTextInput(props: Props): React.Node {
   // Therefore, we ignore selections and pass them through until the selection event has
   // been sent.
   // Note that this mitigation is NOT needed for Fabric.
+  // discovered when upgrading react-hooks
+  // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
This should also let us remove the selective dependency resolution for react eslint rules in the root "package.json"